charts/karmada/_crds/bases/work.karmada.io_works.yaml

```diff
@@ -16,6 +16,9 @@ spec:
   scope: Namespaced
   versions:
   - additionalPrinterColumns:
+    - jsonPath: .spec.workload.manifests[*].kind
+      name: Kind
```

Comment:
The work `Kind` can be a bit confusing, IMOP, `WorkloadKind` feel a little better.

Reply:
How is `WORKLOAD-KIND`? like `CLUSTER-IP` with splited by `-`.
```
NAMESPACE            NAME                              WORKLOAD-KIND        APPLIED   AGE
karmada-es-member1   cert-manager-674b866f4c           Namespace            True      9d
karmada-es-member1   karmada-impersonator-7cbb6bd5c9   ClusterRole          True      9d
```

```
NAMESPACE        NAME                           TYPE           CLUSTER-IP   EXTERNAL-IP                                                     PORT(S)   AGE
default          kubernetes                     ClusterIP      10.96.0.1    <none>                                                          443/TCP   88d
karmada-system   karmada-aggregated-apiserver   ExternalName   <none>       karmada-aggregated-apiserver.karmada-system.svc.cluster.local   <none>    88d
```

Reply:
+1

Reply:
done.